### PR TITLE
Use nginx to cache

### PR DESCRIPTION
**Pull request by Sybrand**
Please also review openshift/nginx.conf (I pushed straight to master to simplify openshift builds)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,22 +104,22 @@ ogr2ogr -f "PostgreSQL" PG:"dbname=tileserv host=localhost user=tileserv passwor
 #### Prepare your openshift environment
 
 ```bash
-# we have docker limits, so pull the images local - then put them in openshift
+# we have docker limits, so pull the pg_tileserv images local - then put them in openshift
```
Comment (conbrad) [inline README.md]: local -> locally

**Review by conbrad — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -104,22 +104,22 @@ ogr2ogr -f "PostgreSQL" PG:"dbname=tileserv host=localhost user=tileserv passwor
...
 
 # push it
-docker push image-registry.apps.silver.devops.gov.bc.ca/e1e498-tools/varnish:latest
 docker push image-registry.apps.silver.devops.gov.bc.ca/e1e498-tools/pg_tileserv:latest
+
+# prepare nginx
+oc -n e1e498-tools new-build nginx~https://github.com/bcgov/wps-vector-tileserver.git --context-dir=openshift --name=nginx-tilecache
 ```
 
 #### Deploy pg_tilserver
```
Comment (andrea-williams) [inline README.md]:
```suggestion
#### Deploy pg_tileserver
```

File: README.md [hunk 3]
```diff
@@ -104,22 +104,22 @@ ogr2ogr -f "PostgreSQL" PG:"dbname=tileserv host=localhost user=tileserv passwor
...
 
 # log in to openshift docker
 docker login -u developer -p $(oc whoami -t) image-registry.apps.silver.devops.gov.bc.ca
 
 # push it
-docker push image-registry.apps.silver.devops.gov.bc.ca/e1e498-tools/varnish:latest
 docker push image-registry.apps.silver.devops.gov.bc.ca/e1e498-tools/pg_tileserv:latest
+
+# prepare nginx
+oc -n e1e498-tools new-build nginx~https://github.com/bcgov/wps-vector-tileserver.git --context-dir=openshift --name=nginx-tilecache
```
Comment (andrea-williams) [inline README.md]: if the aim is to make this generic for all IIT/whoever, should prob remove the reference to our specific openshift namespace

**Review by andrea-williams — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (Sybrand) [reply README.md]: I've changed it to a generic statement and added an 'e.g.' with example values.

**Pull request merged**